src/lv_api_map_v8.h

```diff
@@ -249,6 +249,68 @@ static inline void lv_obj_move_background(lv_obj_t * obj)
 #define lv_image_decoder_built_in_open      lv_bin_decoder_open
 #define lv_image_decoder_built_in_close     lv_bin_decoder_close
 
+/**
```

Comment:
I'm not sure that these should be here, and not in their original "place":
- `lv_snapshot_free`
- `lv_snapshot_take_to_buf`
- `lv_image_buf_set_palette`
- `lv_image_buf_free`

Functions (even if deprecated) should stay in their own component, and be removed from there... otherwise during refactoring a component these "moved" functions will be forgotten.

Reply:
Leaving the deprecated functions split in several headers actually make it easier to be forgotten when refactoring. The day we decides to remove deprecated API, we can simply delete api map directly.

Reply:
Sorry, but I disagree. 
To move out (deprecated) functions from their module/component to somewhere else makes code more **fragmented**. 
The functions and data that belong together should be placed as close as possible (in OO into an object). 
(in C# we just add `[Obsolete]` attribute to methods, but **not moving out** of their object)
The day we decide to remove deprecated functions, we can search for word "obsolete", and remove them from their component. 
As I see the purpose of this `lv_api_map_v8.h` is mainly to have "aliases" to make some v8 code to work with v9, and not the container file for deprecated functions.

Reply:
I think it was my proposal somewhere to move these to `lv_api_map_v8.h` but now I agree with @PGNetHun.

It's better for the users to see the depreciated and the normal functions on one place.  We can keep using `lv_api_map*.h` as a backward compatibility layer.